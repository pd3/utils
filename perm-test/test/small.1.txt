

./perm-test -t test/small.1.tgt.txt -b test/small.1.bg.txt -f test/small.1.fai -c test/small.1.calls.txt -n 1e9
./perm-test -t test/small.1.tgt.txt -b test/small.1.bg.txt -f test/small.1.fai -c test/small.1.calls.txt -n 1e9 --no-bg-overlap

    fai:
        1	5000
    bg:
        1000	1100
        1500	1600
        2000	2100
    tgt:
        1300	1400
        1900	2000

    BG	1000	1100
    TGT	1300	1400
    BG	1500	1600
    TGT	1900	2000
    BG	2001	2100

    calls:
        1350	1350
        1351	1351

    exact: ((1400-1300+1 + 2000-1900+1)*1./(1100-1000+1 + 1600-1500+1 + 2100-2001+1 + 1400-1300+1 + 2000-1900+1))**2 = 0.4007936507936508**2 = 0.1606355505165029
    perm-test:  0.160606787
    err: 2.87635e-05


