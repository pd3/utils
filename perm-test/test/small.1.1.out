BG	1	1000	1100
TGT	1	1300	1400
BG	1	1500	1600
TGT	1	1900	2000
BG	1	2001	2100
MSG	Maximum chromosome randomization error due to length discretization: 0.0e+00%
CALL	TGT	1	1350	1350
CALL	TGT	1	1351	1351
NCALLS	2	0
